packages/git/src/browser/git-repository-tracker.ts

```diff
@@ -84,4 +85,14 @@ export class GitRepositoryTracker {
         return this.onGitEventEmitter.event;
     }
 
+    isInRepository(uri: string): boolean {
```

Comment:
maybe: 
```ts
getPath(uri: URI): string | undefined {
        const repository = this.repositoryProvider.selectedRepository;
        if (!repository) return undefined;

        const path = uri.path.toString();
        const repositoryUri = new URI(repository.localUri);
        const repositoryPath = repositoryUri.path.toString();
        if (!path.startsWith(repositoryPath)) return undefined;

        const relativePath = path.substr(repositoryPath.length);
        return relativePath[0] === '/' ? relativePath.substr(1) : relativePath;
    }
```

It allows to get a relative path and one can check with `!!tracker.getPath(uri)`  whether it is in a repository, i know places where a path is useful

Reply:
Having a counterpart will be nice as well:
```ts
getUri(path: string): URI | undefined {
        const repository = this.repositoryProvider.selectedRepository;
        if (!repository) return undefined;
        return new URI(repository.localUri).resolve(path);
    }
```

Reply:
sounds good, I'll add it.

Reply:
done.